app/src/main/java/it/quip/android/actvitiy/QuipitHomeActivity.java

```diff
@@ -85,7 +85,7 @@ private void updateSidebarMenu() {
         circlesSubMenu.clear();
 
         for (Circle circle : user.getCircles()) {
-            circlesSubMenu.add(0, (int) circle.getUid(), Menu.NONE, circle.getName());
+            circlesSubMenu.add(0, Integer.parseInt(circle.getObjectId()), Menu.NONE, circle.getName());
```

Comment:
Ah you're right. We'll have to come up with a different id system for menu objects.